widget.scss

```diff
@@ -0,0 +1,20 @@
+.widgetContainer {
+	display: grid;
+	grid-template-rows: auto 1fr;
+	grid-template-columns: 100px 100px 100px;
+ 
+	& > .special-widget {
+		border: 1px solid black;
+	}
+}
+
+.widget {
+	margin: 10px;
+	width: 100px;
+	height: 100px;
+	border: 1px solid black
+	
+	& label {
```

Comment:
I wonder if there might be a better choice of semantic html here. `<label>` is meant for html form inputs, if we're not displaying form inputs, then perhaps an `h1-6` or `p` tag would be more suitable?